### ISSUE TITLE
Bullet Surgery don't work, t. Surgeon General

### ISSUE THREAD

**Issue by SplergeBucket** [HippieStationArchive/HippieStation13-1#650]
Reply PM from-Killing Torcher/(Killing Torcher): alright so
Reply PM from-Killing Torcher/(Killing Torcher): I was braindead for a while, I got back and the botanist shot me in the head, quite a few times. A guy saved me and defibed me. He cryo'd me and when I was fixed, I went and gave myself synaptizine, anti-toxins and imidazoline from chemistry. I noticed I was still bleeding from my head. I tried to fix it with gauze, it didn't work. So I concluded it was the bullets. I went to surgery and gave myself bullet surgery on my head, but it didn't work. That's basically it. So I kept bleeding. Eventually another antag came with a gun and killed me.

tl;dr did the surgery for digging for bullets; it listed that he didn't have anything in his head, bleeding didn't stop as a result of the invisible bullets lodged in his skull and he died

**Comment by SplergeBucket** [comment]
Got another response from someone about how it didn't work and that Cryo wasn't working; really needs to be looked into.

**Comment by Carbonhell** [comment]
Unable to reproduce. Do you remember what gun was it?Surgery works just fine with bullets embedded by a glock.

**Comment by Carbonhell** [comment]
If there are no other cases of this issue, i suggest closing this

**Comment by Chronitonity** [comment]
Just stick a "Can Not Reproduce" and keep it open for a bit.
